src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,34 @@
+package duke;
+import duke.exception.DukeException;
+import duke.task.TaskList;
+import java.util.*;
+
+/**
+ * Defines the main logic of the Duke bot.
+ */
+public class Duke {
+
+    public static boolean offBot = false;
```

Comment:
Perhaps you would like to rename this to sound more like a boolean i.e. isOffBot?